### ISSUE TITLE
Version undefined on mac

### ISSUE THREAD

**Issue by stephanieluz** [plentico/plenti#156]
I just updated Plenti to the newest version `brew upgrade plenti`:

<img width="1275" alt="Screen Shot 2021-04-08 at 4 30 28 PM" src="https://user-images.githubusercontent.com/17965122/114092528-fbf3d500-9887-11eb-913a-567fea94270f.png">

If I try running the `plenti --version` command, it returns undefined:

<img width="508" alt="Screen Shot 2021-04-08 at 4 31 00 PM" src="https://user-images.githubusercontent.com/17965122/114092513-f7c7b780-9887-11eb-9ed0-8c397a6e2839.png">

**Comment by jimafisk** [comment]
Have you built Plenti from source on your local machine at all? The version [gets packaged in the binary](https://github.com/plentico/plenti/blob/6a2f146430af3291fae7a6081339e660ba67db27/.goreleaser.yml#L20) during our CI so it's only available if downloaded from our releases (or downloaded with a package manager). Can you try `which plenti` to make sure it's coming from Homebrew?

**Comment by stephanieluz** [comment]
I tried downgrading to the last passing version of the build `brew install plenti@0.4.2` to see if it was working then, but I can't do that:

```
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "plenti@0.4.2".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
```

**Comment by jimafisk** [comment]
It seems like the version was getting assigned on the latest build: `cmd=[go build -ldflags=-s -X plenti/cmd.Version=v0.4.13 -o` (see https://github.com/plentico/plenti/runs/2289216396?check_suite_focus=true#step:10:591)

when I try to build locally:
```
env GOOS=darwin GOARCH=amd64 CGO_ENABLED=1 CC=/home/runner/work/osxcross/target/bin/o64-clang CXX=/home/runner/work/osxcross/target/bin/o64-clang++ go build -ldflags -s
```

I do get a lot of warnings like:
```
ld: warning: object file (/home/jimafisk/go/pkg/mod/rogchap.com/v8go@v0.5.1/deps/darwin_x86_64/libv8.a(crc_folding.o)) was built for newer macOS version (10.10) than being linked (10.9)
```

That's probably unrelated though...

**Comment by jimafisk** [comment]
Ok I think I figured this out, we updated go mod to use full import paths https://github.com/plentico/plenti/pull/143

Previously I could build the version number in a local release like this (this won't work in fish, but should be fine in bash):
```
go build -ldflags="-X 'main.version=$(git describe --tags)'"
```

Now it should be written like this (again running in bash):
```
go build -ldflags="-X 'github.com/plentico/plenti/cmd.Version=$(git describe --tags)'"
```

After it builds you can run `./plenti -v` and it should return `v0.4.13`. I'll fix this in the next release.

**Comment by jimafisk** [comment]
Should be fixed in [v0.4.14](https://github.com/plentico/plenti/releases/tag/v0.4.14)

**Comment by stephanieluz** [comment]
Yup!

<img width="1277" alt="Screen Shot 2021-04-08 at 6 37 28 PM" src="https://user-images.githubusercontent.com/17965122/114104781-898bf080-9899-11eb-95ea-e4ecf5471efe.png">
